integreat_cms/core/utils/machine_translation_api_client.py

```diff
@@ -6,14 +6,26 @@
 
 import logging
 from abc import ABC, abstractmethod
+from html import unescape
 from typing import TYPE_CHECKING
 
+from django.conf import settings
+from django.utils.html import strip_tags
+
 if TYPE_CHECKING:
     from django.db.models.query import QuerySet
     from django.forms.models import ModelFormMetaclass
     from django.http import HttpRequest
 
-    from ..cms.models import Event, Page, POI, Region
+    from ..cms.models import (
```

Comment:
```suggestion
    from ...cms.models import (
```
Weird that mypy did not notice that :thinking: